pkgs/tools/misc/pls/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pls";
+  version = "4.0.2";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "0j0kiv5crbm5zbgpaa17755rf3iga78hnf7nbj001jziqkns580z";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pyyaml
+    rich
+  ];
+
+  meta = with lib; {
+    homepage = "https://dhruvkb.github.io/pls/";
+    description = "pls is a prettier and powerful ls for the pros";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```

see https://github.com/dhruvkb/pls/blob/main/pyproject.toml#L8

---

Interesting, I based the license line on https://github.com/dhruvkb/pls/blob/main/LICENSE, which just says gpl3